app/src/main/java/com/sopt/now/test/presentation/LoginActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.sopt.now.test.presentation
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.sopt.now.databinding.ActivityLoginBinding
+import com.sopt.now.test.data.ApiFactory
+import com.sopt.now.test.data.UserPreference
+import com.sopt.now.test.data.dto.request.RequestLoginDto
+
+class LoginActivity : AppCompatActivity() {
+
+    private val binding by lazy { ActivityLoginBinding.inflate(layoutInflater) }
+    private val viewModel by viewModels<LoginViewModel>()
+    private lateinit var userPreference: UserPreference
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        userPreference = UserPreference(this)
+        ApiFactory.initializeUserPreference(userPreference)
+
+        initObserver()
+        setupLoginButton()
+        setupSignUpTextView()
+    }
+
+    // 로그인
+    private fun setupLoginButton() {
+        binding.btnLogin.setOnClickListener {
+            viewModel.login(getLoginRequestDto())
+        }
+    }
+
+    private fun initObserver() {
+        // 사용자 데이터 저장
+        viewModel.userIdLiveData.observe(this) { userId ->
+            userId?.let {
+                userPreference.saveUserId(userId)
+                moveToMain()
+            }
+        }
+
+        viewModel.liveData.observe(this) { response ->
+            showToast(response.message)
+        }
+    }
+
+    private fun getLoginRequestDto(): RequestLoginDto {
+        val id = binding.etLoginId.text.toString()
+        val password = binding.etLoginPw.text.toString()
+        return RequestLoginDto(
+            authenticationId = id,
+            password = password
+        )
+    }
```

Comment:
```suggestion
    viewModel.login(RequestLoginDto(
            authenticationId = binding.etLoginId.text.toString(),
            password = binding.etLoginPw.text.toString()
        )
    )
```
처럼 추가적인 함수화 대신 위의 리스너에서 바로 활용해도 깔끔할 것 같아요

Reply:
넵! 수정했습니다!